apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -54,24 +54,31 @@ if (NOT CBLAS_FOUND)
   message(STATUS "linear_algebra: No CBLAS header, skipping BLAS benchmarks")
 else()
   foreach(BLAS_VENDOR ${BLAS_VENDORS})
-    string(TOLOWER ${BLAS_VENDOR} NAME)
-    set(TARGET ${NAME}_benchmarks)
-    add_executable(${TARGET}
-      cblas_benchmarks.cpp
-    )
-    target_include_directories(${TARGET} SYSTEM
-      PRIVATE
-       ${CBLAS_INCLUDE_DIR}
-    )
-    string(TOUPPER ${BLAS_VENDOR} DEFINE_SUFFIX)
-    target_compile_definitions(${TARGET} PRIVATE -DUSE_${DEFINE_SUFFIX} ${MISC_DEFINITIONS})
-    target_compile_options(${TARGET} PRIVATE -Wno-error=unused-variable)
-    target_link_libraries(${TARGET}
-      PRIVATE
-       ${BLAS_${NAME}_LIBRARY}
-       ${${BLAS_VENDOR}_EXTRA_LIBS}
+    set(BLA_VENDOR ${BLAS_VENDOR})
```

Comment:
BLA_VENDOR?

Reply:
Same pattern that is used in apps/linear_algebra/CMakeLists.txt

Reply:
(May actually be unused. I can check.)

Reply:
Yeah, something is still weird. Investigating.

Reply:
BLA_VENDOR is a thing; see https://cmake.org/cmake/help/v3.0/module/FindBLAS.html